### DISCUSSION TITLE
Grid convergence

### DISCUSSION THREAD

**Discussion by nahidrahmati** [UvaCsl/HemoCell#26] [General discussion]
Hi, 
I have run a setup with dx=0.5 micron and RBC with 600 triangles per cell. When I increase dx to study about the grid convergence, Some RBC cells are removed in the initial iteration. Can you please explain why some of them are not readable when the grid increase?

**Comment by gzavo** [comment]
Hi @nahidrahmati,
While it is difficult to tell it for sure from this, if you have walls, their location also changes with the dx change. It can be that cells are now too close or even overlapping with the wall, so they are getting deleted.
